### ISSUE TITLE
Warning: Blocking call detected in HA 2024.9.0rc

### ISSUE THREAD

**Issue by mvdwetering** [mvdwetering/huesyncbox#118]
**Describe the bug**

Following warning shows up in HA logs on 2024.9.0rc

```
Detected blocking call to load_verify_locations with args (<ssl.SSLContext object at 0x7f0ea3bae850>, None, None, '\n-----BEGIN CERTIFICATE----- <cut for clarity>-----END CERTIFICATE-----\n') inside the event loop by custom integration 'huesyncbox' at custom_components/huesyncbox/__init__.py, line 52: api = aiohuesyncbox.HueSyncBox( (offender: /usr/lib/python3.12/ssl.py, line 708: context.load_verify_locations(cafile, capath, cadata)), please create a bug report at https://github.com/mvdwetering/huesyncbox/issues For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#load_verify_locations Traceback (most recent call last):

File "/home/michel/hass_dev/venv/bin/hass", line 8, in <module> sys.exit(main())
File "/home/michel/hass_dev/homeassistant/__main__.py", line 209, in main exit_code = runner.run(runtime_conf)
File "/home/michel/hass_dev/homeassistant/runner.py", line 189, in run return loop.run_until_complete(setup_and_run_hass(runtime_config))
File "/usr/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete self.run_forever()
File "/usr/lib/python3.12/asyncio/base_events.py", line 641, in run_forever self._run_once()
File "/usr/lib/python3.12/asyncio/base_events.py", line 1987, in _run_once handle._run()
File "/usr/lib/python3.12/asyncio/events.py", line 88, in _run self._context.run(self._callback, *self._args)
File "/home/michel/hass_dev/homeassistant/setup.py", line 165, in async_setup_component result = await _async_setup_component(hass, domain, config)
File "/home/michel/hass_dev/homeassistant/setup.py", line 461, in _async_setup_component await asyncio.gather(
File "/home/michel/hass_dev/homeassistant/setup.py", line 463, in <genexpr> create_eager_task(
File "/home/michel/hass_dev/homeassistant/util/async_.py", line 45, in create_eager_task return Task(coro, loop=loop, name=name, eager_start=True)
File "/home/michel/hass_dev/homeassistant/config_entries.py", line 752, in async_setup_locked await self.async_setup(hass, integration=integration)
File "/home/michel/hass_dev/homeassistant/config_entries.py", line 604, in async_setup result = await component.async_setup_entry(hass, self)
File "/home/michel/hass_config/custom_components/huesyncbox/__init__.py", line 52, in async_setup_entry api = aiohuesyncbox.HueSyncBox(
```

**To Reproduce**
1. Install 2024.9.0rc
2. Start Home Assistant
3. Warning appears in logs

**Expected behavior**
No warnings

**Diagnostics**
If possible add diagnostics data from HA. The "Download diagnostics" option can be found in the vertical 3 dots menu (⋮) of the integration.

Please add the file itself (drag-and-drop in this box). Do not, copy-paste the contents here as it will make the issue unreadable

**Logging**
If there are errors in the Home Assistant logs please add them here.

See description

**Environment**
Provide details about the versions you are using, which helps to reproduce and find the issue quicker.

- Home Assistant Core version:  2024.9.0rc branch
- Home Assistant operating environment Core
- Huesyncbox integration version: 2.2.3
- Last working version (if known): N.A.

**Additional info**
Add any other info about the problem here.

NA

**Comment by mvdwetering** [comment]
I have a fix in the works, but running into (what seems to be) a bug in library I depend on. 
So will have to wait for that to get resolved.

**Comment by mvdwetering** [comment]
Got closed automatically, but not released yet

**Comment by mvdwetering** [comment]
Is fixed in release V2.2.4
